emission/core/wrapper/suggestion_sys.py

```diff
@@ -38,18 +39,24 @@
 """
 Checks if conf files exists or not. The conf files will be given to the user through request.
 """
-assert os.path.exists(google_maps_path), "File %s not found" % google_maps_path
-assert os.path.exists(yelp_json_path), "File %s not found" % yelp_json_path
-assert os.path.exists(nominatim_path), "File %s not found" % nominatim_path
+try:
+    google_maps_json = open('conf/net/ext_service/googlemaps.json', 'r')
```

Comment:
why do you need these? the `conf/net/ext_service` files are already loaded in `nominatim.py`, aren't they? Please remove

Reply:
Removed the nominatim conf files